src/torchmetrics/detection/modified_panoptic_quality.py

```diff
@@ -0,0 +1,210 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Any, Collection, Optional, Sequence, Union
+
+import torch
+from torch import Tensor
+
+from torchmetrics.functional.detection._panoptic_quality_common import (
+    _get_category_id_to_continuous_id,
+    _get_void_color,
+    _panoptic_quality_compute,
+    _panoptic_quality_update,
+    _parse_categories,
+    _prepocess_inputs,
+    _validate_inputs,
+)
+from torchmetrics.metric import Metric
+from torchmetrics.utilities.imports import _MATPLOTLIB_AVAILABLE
+from torchmetrics.utilities.plot import _AX_TYPE, _PLOT_OUT_TYPE
+
+if not _MATPLOTLIB_AVAILABLE:
+    __doctest_skip__ = ["ModifiedPanopticQuality.plot"]
+
+
+class ModifiedPanopticQuality(Metric):
+    r"""Compute `Modified Panoptic Quality`_ for panoptic segmentations.
+
+    The metric was introduced in https://arxiv.org/abs/1905.01220 Section 4, and is an adaptation of the original
```

Comment:
```suggestion
    The metric was introduced in `Seamless Scene Segmentation paper`_, and is an adaptation of the original
```
please add https://arxiv.org/abs/1905.01220 to https://github.com/Lightning-AI/metrics/blob/master/docs/source/links.rst instead

Reply:
[68ac867](https://github.com/Lightning-AI/metrics/pull/1627/commits/68ac86737e34db0592ad73347640ca5978d08174)